config-connector/solutions/sql/kpt/mysql-public/README.md

```diff
@@ -0,0 +1,49 @@
+MySQL Public
+==================================================
+# NAME
+  mysql-public
+# SYNOPSIS
+  Config Connector compatible yaml files to configure a public mySQL cluster
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/sql/kpt/mysql-public mysql-public
+  ```
+# SETTERS
+|     NAME      |         VALUE         |     SET BY      |      DESCRIPTION      | COUNT |
+|---------------|-----------------------|-----------------|-----------------------|-------|
+| database-name | mysql-public-database | package-default | name of SQL database  | 1     |
+| instance-name | mysql-public-solution | package-default | name of SQL instance  | 3     |
+| password      | ${PASSWORD?}          | PLACEHOLDER     | password for SQL user | 1     |
+| username      | ${USERNAME?}          | PLACEHOLDER     | name of SQL user      | 1     |
```

Comment:
In SQL, username is not a security concern and let's just use a default username, and remove it from the setters. Can be something like "default", "test", etc.